sci-biology/ensembl/ensembl-9999.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit git-r3 perl-functions
+
+DESCRIPTION="The Ensembl Perl API and SQL schema"
+HOMEPAGE="http://www.ensembl.org/info/docs/api/"
+SRC_URI=""
+EGIT_REPO_URI="https://github.com/Ensembl/${PN}.git"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+IUSE="+variation +funcgen +compara +io"
+
+DEPEND="sci-biology/bioperl"
+RDEPEND="${DEPEND}
+	variation? ( sci-biology/ensembl-variation )
+	funcgen? ( sci-biology/ensembl-funcgen )
+	compara? ( sci-biology/ensembl-compara )
+	io? ( sci-biology/ensembl-io )"
+
+src_install() {
+	perl_set_version
+# the line below should work but doesn't (double prefix)
+#	insinto "${VENDOR_LIB}/EnsEMBL"
```

Comment:
Well I guess it should be clarified but it is all down to how you are supposed to use the variable.  If you are supposed to use it in `insinto` that's a bug otherwise it is something you have to careful of. It being prefixed is useful if you have to point the build system to the location of the perl install for example _ during configuration or build rather than install I mean.